### PR TITLE
[PR-7044] Support multiple site slugs in Talkable Android SDK

### PR DESCRIPTION
**Pull request by dgtv**
Demos:
http://void-docs.talkable.com/android_sdk/getting_started.html
http://void-docs.talkable.com/android_sdk/advanced.html
http://void-docs.talkable.com/android_sdk/upgrade.html

### CODE REVIEW

File: source/android_sdk/advanced.rst [hunk 1]
```diff
@@ -4,51 +4,107 @@
...
+         @Override
+         public void onCreate() {
+             super.onCreate();
+             Talkable.initialize(this, "your-default-site-slug");
+         }
+     }
+
+   .. note::
+
+     You can set site slug at any time after initilization in the following way:
```
Comment (zhuravel) [inline source/android_sdk/advanced.rst]: initilization → initi**a**lization

File: source/android_sdk/advanced.rst [hunk 2]
```diff
@@ -4,51 +4,107 @@
...
+
+     .. code-block:: java
+
+       Talkable.setSiteSlug(context, "some-site-slug");
+
+     Make sure to add credentials for this site inside the manifest file.
+     Otherwise an exception will be raised.
```
Comment (zhuravel) [inline source/android_sdk/advanced.rst]: Add a comma.
> Otherwise,

File: source/android_sdk/advanced.rst [hunk 3]
```diff
@@ -4,51 +4,107 @@
 Advanced Usage
 ==============
 
+Using multiple site slugs
+-------------------------
+
+To use multiple site slugs inside your application you have to do next steps:
```
Comment (zhuravel) [inline source/android_sdk/advanced.rst]: Add a comma.
> To use multiple site slugs inside your application, you have to...

File: source/android_sdk/advanced.rst [hunk 4]
```diff
@@ -4,51 +4,107 @@
...
-    <activity android:name=".MyFragmentActivity" />
+   .. note::
 
-.. note::
-   You can just override TalkableOfferFragment and use default TalkableActivity from Talkable SDK.
-   In this case you shouldn't change the manifest
-   (if you did steps from :ref:`Getting Started <android_sdk/getting_started>` section).
+      You can just override ``TalkableOfferFragment`` and use default
+      ``TalkableActivity`` from Talkable SDK.
+      In this case you shouldn't change the manifest
```
Comment (zhuravel) [inline source/android_sdk/advanced.rst]: Add a comma.
> In this case,

File: source/android_sdk/getting_started.rst [hunk 5]
```diff
@@ -13,72 +13,117 @@ Installation
...
+     - Select site and go to **Dashboard** |rarr| **Settings** |rarr| **Site Settings**.
+       Find **Integration settings** section and there you will see your API Keys.
+       Use only the public key in your application.
 
-.. code-block:: xml
+.. _deep_linking_scheme:
 
-  <intent-filter>
-      <action android:name="android.intent.action.VIEW" />
+5. Add deep linking schema handler into your main activity entry.
```
Comment (zhuravel) [inline source/android_sdk/getting_started.rst]: So, is it schem**e** or schem**a**?

File: source/android_sdk/advanced.rst (hunk 4 above)
Comment (dgtv) [inline source/android_sdk/advanced.rst]: Thank you

**Comment by dgtv** [comment]
@bogdan could you also take a look please?

**Pull request merged**